src/style.css

```diff
@@ -1 +1,61 @@
 /* Styles go here */
+html {
+  --links-color: #00acdc;
```

Comment:
Move to :root selector.

Reply:
> Move to :root selector.
I didn't understand. HTML is just the root for all tags